test/helpers/user_blocks_helper_test.rb

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+require "test_helper"
+include ApplicationHelper
```

Comment:
Does that actually do anything there? What class is that going to include into?

I assume that's for `friendly_date` so if it working placed there then I think it's only by some weird accident and the correct location in inside the `UserBlocksHelperTest` class definition.